webapp/src/pages/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,81 @@
+.login {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  min-height: 40rem;
+  overflow: scroll;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  grid-template-columns: repeat(var(--column-number), 1fr);
+  grid-template-rows: auto 1fr auto;
+  grid-template-areas:
+    'title title title title'
+    'main main main main'
+    'end end end end';
+}
+
+.login-title > *,
+.login-subtitle > * {
+  margin: 0;
+}
+
+.login-title {
+  grid-area: title;
+}
+
+.login-container {
+  grid-area: main;
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem var(--column-gap);
+  align-items: center;
+}
+
+.login-form .inputs-container > .input {
+  margin-bottom: 0.875rem;
+  width: 15rem;
+}
+
+.login-form .inputs-container > p:empty::before {
+  content: '';
+  display: inline-block;
+}
+
+.final-text {
+  display: flex;
+  gap: 0.3rem;
+}
+
+.final-text a {
+  text-decoration: none;
+}
+
+.login .landing-animation {
+  display: none;
+}
+
+/* Styles for viewports wider than 768 pixels. */
+@media (min-width: 768px) {
+  .login {
+    min-height: 20rem;
+    padding: 4rem var(--padding-sides) 8rem var(--padding-sides);
```

Comment:
maybe an `@import` between this css and the landing css would prevent the whole block of duplicated code we have?